content/descriptor.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package content
+
+import (
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// MediaTypeOctetStream is the default mediaType used when no
+// media type is specified.
+const MediaTypeOctetStream string = "application/octet-stream"
+
+// NewDescriptorFromBytes returns a descriptor, given the content and media type.
+// If no media type is specified, MediaTypeOctetStream will be used.
+func NewDescriptorFromBytes(content []byte, mediaType string) ocispec.Descriptor {
```

Comment:
can we have
```suggestion
func NewDescriptorFromBytes(mediaType string, content []byte) ocispec.Descriptor {
```
just be consistent with the descriptor?

Reply:
changed accordingly